plugins/text-editor-resources/src/components/ReferenceInput.svelte

```diff
@@ -84,7 +84,7 @@
     insertTemplate: (name, markup) => {
       editor?.insertMarkup(markup)
     },
-    insertTable (options: { rows?: number, cols?: number, withHeaderRow?: boolean }) {
+    insertTable(options: { rows?: number; cols?: number; withHeaderRow?: boolean }) {
```

Comment:
To me it looks like someones IDE is not configured to format the code properly. Do we need some kind of pipeline to reject unformatted code? Or maybe husky to format the code before push?

Reply:
There is no husky in that project. 
The file is saved with the prettier-eslint extension. So I think that my new one is right.

Reply:
Yeah, I meant that it should be added otherwise these kind of unnecessary changes will be pushed and someone has to code review these kind of changes while we can prevent it simply by adding something like husky. I do not know how ususally thing are done here but if it were up to me I would have created a task called add husky or at least do some R&D around it to find out whether it worth the effort or not.